ionoscloud/resource_server.go

```diff
@@ -50,8 +50,10 @@ func resourceServer() *schema.Resource {
 				Computed: true,
 			},
 			"availability_zone": {
-				Type:     schema.TypeString,
-				Optional: true,
+				Type:         schema.TypeString,
+				Optional:     true,
+				Computed:     true,
+				ValidateFunc: validation.All(validation.StringInSlice([]string{"AUTO", "ZONE_1", "ZONE_2", "ZONE_3"}, true)),
```

Comment:
From what I see in the swagger file, for **ServerProperties** we have only _AUTO, ZONE_1, ZONE_2,_ and not _ZONE_3._ Not sure if ZONE_3 works on the API, but we could address this inconsistency and update here or on the swagger file.

Reply:
I checker now and ZONE_3 isn't